.github/workflows/build-jsp.yml

```
@@ -11,12 +15,16 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
+    - name: Get the CLDR_REF from pom.xml
+      id: cldr_ref
+      run: pip install yq && echo "CLDR_REF="$(xq -r '.project.properties["cldr.version"]' < pom.xml | cut -d- -f3) >> $GITHUB_OUTPUT && cat ${GITHUB_OUTPUT}
```

```suggestion
      run: echo "CLDR_REF="$(mvn help:evaluate -Dexpression=cldr.version -q -DforceStdout | cut -d- -f3) >> $GITHUB_OUTPUT && cat ${GITHUB_OUTPUT}
```
I think you can replace the usage of `yq` with the Maven command that gives you the same.  Do you need the `$GITHUB_OUTPUT`, or can you remove that on the assumption that whatever is streamed to standard output will be captured just the same?

---

GITHUB_OUTPUT is for copying the value to another step in the process, so yes needed.

There's a maven command? I thought there might be but couldn't find it.

---

https://docs.github.com/en/actions/learn-github-actions/contexts#example-usage-of-the-steps-context

---

ok now i see it. VS Code showed me your comment but not the suggestion.